arcgis-ios-sdk-samples/Display information/Sketch on the map/SketchViewController.swift

```diff
@@ -16,76 +16,82 @@ import UIKit
 import ArcGIS
 
 class SketchViewController: UIViewController {
-    @IBOutlet private weak var mapView: AGSMapView!
-    @IBOutlet private weak var geometrySegmentedControl: UISegmentedControl!
-    @IBOutlet private weak var undoBBI: UIBarButtonItem!
-    @IBOutlet private weak var redoBBI: UIBarButtonItem!
-    @IBOutlet private weak var clearBBI: UIBarButtonItem!
-    
-    private var sketchEditor: AGSSketchEditor!
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Configure the sketch editor.
+            sketchEditor = AGSSketchEditor()
+            mapView.sketchEditor = sketchEditor
+            
+            // Start a default sketch editor with the polyline creation mode.
+            sketchEditor.start(with: nil, creationMode: .polyline)
+            
+            // Set the map.
+            mapView.map = AGSMap(basemapStyle: .arcGISLightGrayBase)
+            mapView.interactionOptions.isMagnifierEnabled = true
+            // Set the viewpoint.
+            mapView.setViewpoint(AGSViewpoint(targetExtent: AGSEnvelope(xMin: -10049589.670344, yMin: 3480099.843772, xMax: -10010071.251113, yMax: 3512023.489701, spatialReference: .webMercator())))
+        }
+    }
+    @IBOutlet var addBarButtonItem: UIBarButtonItem!
+    @IBOutlet var undoBBI: UIBarButtonItem!
+    @IBOutlet var redoBBI: UIBarButtonItem!
+    @IBOutlet var clearBBI: UIBarButtonItem!
     
+    var sketchEditor: AGSSketchEditor!
```

Comment:
If it always exists, 
```suggestion
    var sketchEditor = AGSSketchEditor()
```
and remove the line in map view didSet.